pkg/controller/servicebindingrequest/secret.go

```diff
@@ -27,12 +28,19 @@ func (s *secret) buildResourceClient() dynamic.ResourceInterface {
 
 // createOrUpdate will take informed payload and either create a new secret or update an existing
 // one. It can return error when Kubernetes client does.
-func (s *secret) createOrUpdate(payload map[string][]byte) (*unstructured.Unstructured, error) {
+func (s *secret) createOrUpdate(payload map[string][]byte, secretOwnerReference v1.OwnerReference) (*unstructured.Unstructured, error) {
 	logger := s.logger.WithValues("Namespace", s.ns, "Name", s.name)
+	reference := metav1.OwnerReference{
+		Name:       secretOwnerReference.Name,
+		UID:        secretOwnerReference.UID,
+		Kind:       secretOwnerReference.Kind,
+		APIVersion: secretOwnerReference.APIVersion,
```

Comment:
We should also set the controller value to true, 
controller:true
the controller value within ownerReferences is there to tell the user that the ownerRef was set via a controller and not a human.

Reply:
done as a part of this commit 3a5341c